Example/Example/AppDelegate.swift

```diff
@@ -266,4 +265,31 @@ private extension AppDelegate {
             }
         }
     }
+    
+    func conditionalRules() {
+        let siren = Siren.shared
+        siren.delegate = self
+        siren.presentationManager = PresentationManager(nextTimeButtonTitle: "My Next", skippableAfter: 2)
```

Comment:
"My Next" is a copy we override in the Later app right @nvprokofiev ?

Reply:
yes, this is just an example. We override it in the app to "Next"
The default value was "Not now" so I had to add customization.